lib/handlers/quote/schema/quote-schema.ts

```diff
@@ -62,6 +62,16 @@ export const QuoteQueryParamsJoi = Joi.object({
   unicornSecret: Joi.string().optional(),
   intent: Joi.string().valid('quote', 'swap', 'caching', 'pricing').optional().default('quote'),
   enableFeeOnTransferFeeFetching: Joi.boolean().optional().default(false),
+  portionBips: Joi.string()
+    .pattern(/^[0-9]+$/)
+    .max(5) // 0% < portionBips < 100%
+    .optional(),
+  portionAmount: Joi.string()
+    .pattern(/^[0-9]+$/)
+    .optional(),
+  portionRecipient: Joi.string()
```

Comment:
can just use the same JOI as tokenIn address above:
`Joi.string().alphanum().max(42).optional()`